ev3dev/optional.py

```diff
@@ -0,0 +1,692 @@
+"""
```

Comment:
If we are moving them now and long term we want them to be in sound.py and screen.py files (I like that idea) why not go ahead and put them there now?

Reply:
To be honest, I put them in the same file because the existing `framebuffer.py` was annoying me :laughing: Although, I am not entirely sure how I want to organize the platform check; I would prefer that the logic to compare implementation names is centralized in one location. Thoughts?